cmd/root.go

```diff
@@ -71,6 +71,11 @@ func persistentValidateFunc(rootCommand *RootCmd, rootOpts *RootOptions) func(c
 		// Avoid sensitive info into default values help line
 		rootCommand.StripSensitive()
 
+		// We just use ubuntu internally
+		if strings.HasPrefix(rootOpts.Target, "ubuntu") {
```

Comment:
Ok, we basically support any "ubuntu-*" using just "ubuntu" internally.
Note that this means we also support "ubuntu-garbage", ie any string after "ubuntu", but i think it is pretty fair indeed.